cfme/fixtures/pytest_selenium.py

```diff
@@ -1571,37 +1566,37 @@ def deselect_by_value(select_element, val):
     return _sel_desel(select_element, lambda s: ByValue(value(s)), 'deselect_by_value', val)
 
 
-@multidispatch
-def deselect(loc, o):
-    raise NotImplementedError('Unable to select {} in this type: {}'.format(o, loc))
+def deselect(loc, vals_or_texts):
+    """
 
+    Args:
+        loc: A locator, expects either a string, WebElement, tuple.
+        val_or_text: either a single or an iterable of option values or description texts
 
-@deselect.method((object, ByValue))
-def _deselect_val(loc, val):
+    """
     # Do not "cast" the loc unless it is needed
-    from cfme.web_ui import AngularSelect
-    if type(loc) in {Select, AngularSelect}:
-        l = loc
+    loc = _locator_for_select(loc)
+
+    if isinstance(vals_or_texts, Iterable):
```

Comment:
```python
and not isinstance(vals_or_texts, basestring)
```
since

```python
isinstance("asdf", Iterable)  # => True
```

Sorry for not spotting that before.